examples/sandbox/src/App.js

```diff
@@ -230,6 +231,11 @@ export default function App() {
                         label="Acres"
                         view={SingleSelectFacet}
                       />
+                      <Facet
```

Comment:
There is already a `Facet` for this field further up in this file. Can you remove this and change the view of the existing facet?

Reply:
Sure!